core/tests/puppeteer-acceptance-tests/spec/helpers/reporter.js

```diff
@@ -0,0 +1,249 @@
+const util = require('util');
```

Comment:
Missing copyright notice? (This code is original, right?)

Also, drop all stuff you don't need.

Reply:
Ummm, Some of it is taken from the original jasmine reporter which was in `node_modules/jasmine/lib/reporter/console-reporter.js`. I think i should add notice, right ?